### ISSUE TITLE
SSL Proxy List can't be edited.

### ISSUE THREAD

**Issue by donileo** [ProxymanApp/Proxyman#811]
### Proxyman version? (Ex. Proxyman 1.4.3)
2.19.0 (21900)

### macOS Version? (Ex. mac 10.14)
11.2.2 (20D80)

### Steps to reproduce 
1. Add a Wildcard Domain to the SSL Proxy List.
2. Double click on it to edit it.
3. Verify nothing happens.

### Expected behavior
Double clicking on an entry should allow user to edit the domain so that they don't have to go through the process of deleting it and then having to re-add it as that would be super annoying.

**Comment by NghiaTranUIT** [comment]
Yes, I totally agree. Each entry should be able to edit. I will add this feature in the next release 😄

**Comment by sleeve** [comment]
@NghiaTranUIT This enhancement was never implemented. Any plans on getting it added soon?

**Comment by NghiaTranUIT** [comment]
Ah nice @sleeve, I'm adding this ticket to the next release. Definitely working on it today 👍

**Comment by NghiaTranUIT** [comment]
@sleeve @isadon it's ready: https://download.proxyman.io/beta/Proxyman_4.16.0_Edit_SSL_Proxying_Entry.dmg

We can simply:
- Double click
- Command + Enter 

to edit the current domain/app 👍

**Comment by sleeve** [comment]
Thanks! The fix looks good on the above beta test build.
